### PR TITLE
LYNX-152: Documentation for attributesMetadata query

### PR DESCRIPTION
**Pull request by eliseacornejo**
## Purpose of this pull request

This pull request (PR) is to add the documentation for GraphQl attributesMetadata query

This PR is related to this jira ticket: [LYNX-152](https://jira.corp.adobe.com/browse/LYNX-152) and also it is related to this jira ticket: [LYNX-160](https://jira.corp.adobe.com/browse/LYNX-160)

## Affected pages

- New page: graphql/schema/eav/queries/custom-attribute-metadata-v2/
- Deprecated query: https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/attributes-metadata/ and https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/custom-attribute-metadata/

whatsnew
Added the [`customAttributeMetadataV2` query](https://developer.adobe.com/commerce/webapi/graphql/schema/eav/queries/custom-attribute-metadata-v/) and deprecated the `attributesMetadata` and `customAttributeMetadata` queries.

### CODE REVIEW

**Comment by keharper** [comment]
@eliseacornejo It will take a few days to get to this PR. 

I know that a version of this query was packaged with PWA and is documented at https://developer.adobe.com/commerce/webapi/graphql/schema/store/queries/attributes-metadata/

We need to find a way to differentiate these two identically named queries, and provide guidance for anyone who has already installed it.

Also, I placed the original `attributesMetadata` under Schema > Store , because I was trying to avoid creating numerous areas under Schema that each have only one or two queries/mutations. Can this query be moved to the same query as the old version?

**Comment by keharper** [comment]
Maybe this query needs to be renamed to `attributesMetadataV2`.

**Comment by eliseacornejo** [comment]
@keharper Yes, you are right, we are going to rename this query.  So I will be updating this PR, after having this. Also I would deprecate the other one in PWA, so we need to add something on that query that it is deprecated. 
For the placement, I don't think the best place is to be under store as it is not a query totally related to store, this is a query that can retrieve attributes for different entities like catalog_product, customer, customer_address, this is why we created a new area.

**Review by keharper — CHANGES_REQUESTED** [review]
Please back out changes to the `static/graphql-api/index.html` file. We have automation that will generate this file when we're ready to publish the autogenerated schema docs.

**Review by keharper — COMMENTED** [review]
@eliseacornejo I talked to Cris Partica about the placement of the new query. This query and the form query isn't limited to EAV. Instead, please create a `attributes` directory, and move the file there. Also, update the left navigation from Eav to Attributes. 

I'm going to move the deprecated queries immediately after merging this PR.

File: src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+---
+
+# customAttributeMetadataV2 query
+
+The `customAttributeMetadataV2` query retrieves metadata for the specified `entity_type` and `attribute_code` pairs.
+
+This query replaces [`attributesMetadata`](../../store/queries/attributes-metadata.md) 
+and [`customAttributeMetadata`](../../store/queries/custom-attribute-metadata.md). 
+
+Some of the differences of this new query are:
```
Comment (keharper) [inline src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md:12]:
```suggestion
This new query has several features that were not available in the deprecated queries:

```
(Add a blank line after the colon)

File: src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+- Can be extended by all entity types.
```
Comment (keharper) [inline src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md:13]:
```suggestion
-  All entity types can extend this query.
```

File: src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+# customAttributeMetadataV2 query
+
+The `customAttributeMetadataV2` query retrieves metadata for the specified `entity_type` and `attribute_code` pairs.
+
+This query replaces [`attributesMetadata`](../../store/queries/attributes-metadata.md) 
+and [`customAttributeMetadata`](../../store/queries/custom-attribute-metadata.md). 
+
+Some of the differences of this new query are:
+- Can be extended by all entity types. 
+- This provides access to all properties of eav attributes.
```
Comment (keharper) [inline src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md:14]:
```suggestion
-  The query provides access to all properties of EAV attributes.
```

File: src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+- It introduced uid information for the attributes.
```
Comment (keharper) [inline src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md:15]:
```suggestion
-   `uid` values are now available.

```

Add a blank line after the last bullet. Also make sure there are no blank spaces at the end any of the lines.

**Pull request merged**